Makefile

```diff
@@ -26,7 +26,7 @@ api-start: ## Start the API
 
 .PHONY: api-test
 api-test: ## Test the API
-	@docker compose run --rm api pytest --no-cov-on-fail --cov -vvv -s
+	@docker compose run --rm api pytest --no-cov-on-fail --cov -vvv --create-db -s
```

Comment:
Did you mean to write `--reuse-db`?

Reply:
`--reuse-db` is in the pyproject.toml file (pytest.addopts) i removed `--create-db` from the makefile here: https://github.com/adfinis/Outdated/pull/141/commits/512ac4725ab1cc163c1e5ac19418711bdd58ba29